CONTRIBUTING.md

```diff
@@ -1,64 +1,94 @@
-approved it, but you must do it before we can put your code into our codebase.
-Before you start working on a larger contribution, you should get in touch with
-us first with your idea so that we can help out and possibly guide
-you. Coordinating up front makes it much easier to avoid frustration later on.
+***NOTE***: Only original source code from you and other people that have signed the CLA can be accepted into the main repository.
 
+### Community Guidelines
 
-### Code Reviews
+This project follows
+[Google's Open Source Community Guidelines](https://opensource.google/conduct/).
 
-All submissions, including submissions by project members, require review. We
-currently use [Gerrit Code Review](https://www.gerritcodereview.com/) for this
-purpose. Currently, team-member submissions go through private reviews, and
-external submissions go through public reviews.
+### Contributing code
 
+If you have improvements to Cobalt, send us your pull requests! For those
+just getting started, Github has a
+[how to](https://help.github.com/articles/using-pull-requests/).
 
-## Submission Process
+Cobalt team members will be assigned to review your pull requests. Once the
+pull requests are approved and pass continuous integration checks, a Cobalt
+team member will merge the pull request.
 
-We admit that this submission process is currently not completely optimized to
-make contributions easy, and we hope to make improvements to it in the
-future. It will always include some form of signing the CLA and submitting the
-code for review before merging changes into the Cobalt master tree.
+### Contribution guidelines and standards
 
-  1. Ensure you or your company have signed the appropriate CLA (see "Before You
-     Contribute" above).
-  1. Rebase your changes down into a single git commit.
-  1. Run `git clang-format HEAD~` to apply default C++ formatting rules,
-     followed by `git commit -a --amend` to squash any formatting changes
-     into your commit.
-  1. Run `git push origin HEAD:refs/for/master` to upload the review to
-     [Cobalt's Gerrit instance](https://cobalt-review.googlesource.com/).
-  1. Someone from the maintainers team will review the code, putting up comments
-     on any things that need to change for submission.
-  1. If you need to make changes, make them locally, test them, then `git commit
-     --amend` to add them to the *existing* commit. Then return to step 2.
-  1. If you do not need to make any more changes, a maintainer will integrate
-     the change into our private repository, and it will get pushed out to the
-     public repository after some time.
+Before sending your pull request for
+[review](https://github.com/tensorflow/tensorflow/pulls),
+make sure your changes are consistent with the guidelines and follow the
+Cobalt coding style.
+
+#### General guidelines and philosophy for contribution
+
+*   Include unit tests when you contribute new features, as they help to a)
```

Comment:
I would do the same here with another level of unordered list if possible. Just make it easier to read.